### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1233] [Galaxy Trivia] [answered]
```
1  23  124  1251  26127  128129  ?
```

**Comment by peterbaccaro** [comment, marked as answer]
The next number in the sequence is **1301311**.

**Reasoning**

If you remove the spaces you get:
- **123**124**125**126**127**128**129**

So the next numbers in the sequence would be:
- 130**131**132

Each number in the sequence increases by one digit so the number will be 7 digits long.

The next number is therefore **1301311**.
